lib/wallet/wallet.js

```diff
@@ -4925,6 +4925,35 @@ class Wallet extends EventEmitter {
   static isWallet(obj) {
     return obj instanceof Wallet;
   }
+
+  /**
+   * Clear a cache.
+   * @param {String} cacheName
+   * @param {String | undefined} cacheKey
+   */
+
+   clearCache(cacheName, cacheKey) {
+      var cache;
+      switch(cacheName) {
```

Comment:
style nit: switches in javascript are very error prone, missing some of the protections that languages like go give you, would prefer if/else if/else